fe/fe-core/src/main/java/com/starrocks/rpc/PBackendService.java

```diff
@@ -90,5 +98,17 @@ public interface PBackendService {
     @ProtobufRPC(serviceName = "PBackendService", methodName = "exec_short_circuit",
```

Comment:
PBackendService will be obsolete，use PInternalService instead

Reply:
done